server/sonar-web/package.json

```diff
@@ -46,6 +46,8 @@
   },
   "devDependencies": {
     "@types/classnames": "2.2.0",
+    "@types/d3-array": "1.2.1",
+    "@types/d3-scale": "1.0.10",
```

Comment:
We already have different versions of those in the dependencies, is it on purpose ?